account_banking_nl_ing_mt940/account_banking_nl_ing_mt940.py

```diff
@@ -102,8 +113,8 @@ def handle_tag_86(self, cr, data):
             self.current_transaction.remote_owner_city = subfields['ORDP'][3]
 
         if 'REMI' in subfields:
-            self.current_transaction.message = '/'.join(
-                filter(lambda x: bool(x), subfields['REMI']))
+            self.current_transaction.message = (
+                '/'.join([x for x in subfields['REMI'] if x]))
```

Comment:
The beauty of the expression you replaced is that it doesn't create a temporary copy of the list. You can have the same effect if you change the list comprehension to a generator expression.

TLDR: remove outermost square brackets